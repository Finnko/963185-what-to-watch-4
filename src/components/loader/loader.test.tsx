import * as React from 'react';
import * as renderer from 'react-test-renderer';
import Loader from './loader';

const props = {
  style: {
    position: `absolute`,
  }
};

describe(`Loader component render correctly`, () => {
  it(`Should Loader component small size render correctly`, () => {
    const tree = renderer
      .create(
          <Loader
            size={`small`}
            {...props}
          />).toJSON();

    expect(tree).toMatchSnapshot();
  });

  it(`Should Loader component medium size render correctly`, () => {
    const tree = renderer
      .create(
          <Loader
            size={`medium`}
            {...props}
          />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
