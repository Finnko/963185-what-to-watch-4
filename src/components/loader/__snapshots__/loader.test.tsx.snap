// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader component render correctly Should Loader component medium size render correctly 1`] = `
<div
  className="loader-wrapper"
  style={
    Object {
      "position": "absolute",
    }
  }
>
  <div
    className="loader medium"
  />
</div>
`;

exports[`Loader component render correctly Should Loader component small size render correctly 1`] = `
<div
  className="loader-wrapper"
  style={
    Object {
      "position": "absolute",
    }
  }
>
  <div
    className="loader small"
  />
</div>
`;
